Mage/Command/Factory.php

```diff
@@ -43,14 +43,17 @@ public static function get($commandName, Config $config)
             // try a custom command
             $className = 'Command\\' . $commandName;
 
+            // TODO use a custom exception
             if (!class_exists($className)) {
                 throw new Exception('Command "' . $commandName . '" not found.');
             }
         }
 
         /** @var AbstractCommand $instance */
+        // TODO dependencies like $config should be injected into constructor
```

Comment:
@GodsBoss By PHP packages @andres-montanez meant composer dependencies and I don't see any reason why should we need ncurses for progress bar (if you're talking about #88).
Of course I mistyped my comment and I meant **phar** package :)